collect_app/src/main/java/org/odk/collect/android/database/FormDatabaseMigrator.java

```diff
@@ -45,97 +24,51 @@
 import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.JR_FORM_ID;
 import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.JR_VERSION;
 import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.LANGUAGE;
-import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.LAST_DETECTED_FORM_VERSION_HASH;
 import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.MD5_HASH;
 import static org.odk.collect.android.provider.FormsProviderAPI.FormsColumns.SUBMISSION_URI;
 
-/**
- * This class helps open, create, and upgrade the database file.
- */
-public class FormsDatabaseHelper extends SQLiteOpenHelper {
-    private static final String DATABASE_NAME = "forms.db";
-    public static final String FORMS_TABLE_NAME = "forms";
-
-    public static final int DATABASE_VERSION = 9;
+public class FormDatabaseMigrator {
```

Comment:
I wanted to pull this out so we could have an object that was easier to setup and work with than the `SQLiteOpenHelper` implementations that need `DatabaseContext`.